keyboards/mlego/m65/keymaps/default/keymap.c

```diff
@@ -24,6 +24,17 @@ enum layer_names {
     _ADJ
 };
 
+#ifdef OLED_ENABLE
```

Comment:
If anything, the oled code here, especially since it appears to be the same for each keymap, should be moved to the keyboard's c file (m65.c, or rev#.c) instead of the keymap c file.  And switched to using something like this:
```c
#ifdef OLED_ENABLE
bool oled_task_kb(void) {
    if (!oled_task_user()) { 
        return false; 
    }
    // oled code
    return true;
}
#endif
```

This allows the keyboard to have a default behavior for the board, that will also work on the qmk configurator, and will allow for users to customize the behavior, if desired.